etc/helm/pachyderm/values.yaml

```diff
@@ -523,7 +523,9 @@ pachd:
   #   - repoReader
   #   robot:eve:
   #   - repoWriter
-  pachAuthClusterRoleBindings: {}
+  pachAuthClusterRoleBindings:
```

Comment:
I forgot that we currently ship empty role bindings. I think this is ok - but if a user is currently activating auth via git ops, but is setting cluster role bindings via API, we will be overwriting those. I think we need to make sure this is what we want